src/pages/Main/MainPage.tsx

```diff
@@ -74,11 +43,18 @@ const MainPage = () => {
       />
       <SortDropdown onSubmitDropdown={handleSubmitDropdown} />
       <StyledContentsWrapper>
-        <TravelVideoList
-          videoNumberInPage={videoNumberInPage}
-          currentPageNumber={currentPageNumber}
-          videoList={videoList}
-        />
+        {loading ? (
+          <Loading loading={true} />
+        ) : (
+          <Suspense fallback={<Loading loading={true} />}>
+            <TravelVideoList
+              error={error}
+              videoNumberInPage={videoNumberInPage}
+              currentPageNumber={currentPageNumber}
+              videoList={videoList}
+            />
+          </Suspense>
+        )}
```

Comment:
`loading` 변수에 따라 `Loading`을 보여주는데, false 일 때도 결국 `Suspense` 에서 로딩 화면을 보여주니 하나로 줄여도 되지 않을까요?? 아래처럼요!

```suggestion
        <Suspense fallback={<Loading loading={true} />}>
          <TravelVideoList
            error={error}
            videoNumberInPage={videoNumberInPage}
            currentPageNumber={currentPageNumber}
            videoList={videoList}
          />
        </Suspense>
```

Reply:
저도 이부분이 의문인데.. 로딩 조건을 주지 않으면, fallback일 때 로딩 컴포넌트가 뜨지 않고, TravelVideoList error가 전달되어서 

 ```
 if (showNoDataMessage) {
      return (
        <StyledContentsWrapper>
          <StyledErrorMessage>
            <p>{NO_DATA_ERROR_MESSAGE}</p>
            <span>{ERROR_MESSAGE}</span>
          </StyledErrorMessage>
        </StyledContentsWrapper>
      );
```
이 코드가 먼저 실행되더라고요..

Reply:
데이터가 오기 전에 이미 `showNoDataMessage`가 전달되는건가 보네용...

Reply:
> 저도 이부분이 의문인데.. 로딩 조건을 주지 않으면, fallback일 때 로딩 컴포넌트가 뜨지 않고, TravelVideoList error가 전달되어서 
> 
> 
> 
>  ```
> 
>  if (showNoDataMessage) {
> 
>       return (
> 
>         <StyledContentsWrapper>
> 
>           <StyledErrorMessage>
> 
>             <p>{NO_DATA_ERROR_MESSAGE}</p>
> 
>             <span>{ERROR_MESSAGE}</span>
> 
>           </StyledErrorMessage>
> 
>         </StyledContentsWrapper>
> 
>       );
> 
> ```
> 
> 이 코드가 먼저 실행되더라고요..